pkg/announce/announce.go

```diff
@@ -104,13 +111,20 @@ func CreateForRelease(opts *Options) error {
 		changelog = opts.changelogHTML
 	}
 
+	logrus.Infof("Trying to get the Go version used to build %s...", opts.tag)
+	goVersion, err := getGoVersion(opts.tag)
+	if err != nil {
+		return err
+	}
+	logrus.Infof("Found the following Go version: %s", goVersion)
```

Comment:
@puerco Maybe I should have put this in the `getGoVersion` function?

Reply:
Oh sorry I did not see it in the log, that's why I left the comment:
```
Step #2: INFO[2020-12-04T15:49:03Z] Creating v1.20.0-rc.1 announcement in /workspace/src 
Step #2: INFO[2020-12-04T15:49:03Z] Trying to get the kube-cross version for release-1.20... 
Step #2: INFO[2020-12-04T15:49:03Z] Found the following kube-cross version: v1.15.5-1 
Step #2: INFO[2020-12-04T15:49:04Z] Release announcement created           
```

Reply:
@puerco I've actually added it after your comment. :slightly_smiling_face: